m4_sessions/1708-inning/group_four.md

```diff
@@ -1,9 +1,35 @@
  # Group Four
 
-### Session Title:
+### Session Title: 
+Privacy: What it means for devs and their users
+
 
 ### Session Outcomes: 
+- Students are aware of broader privacy concern implications
+- Students can make informed choices about privacy impacts for their users
+
 
 ### Session Outline:
+*This session is about privacy in the scope of Corporate snooping, not Government Surveillance. We're limiting our scope to allow more targeted discussions.
+
+- Intros
+
+- Activities
+* short video about creepy stuff?
+* maybe use my creepy app?
+
+- Discussions
+* What is privacy? What does it mean to you?
+* What are you willing to give away? Where do you draw the line for what you're willing to give away?
+* What is innocuous data? How does giving away small bits of data add up?
+* How privacy can impact your app use
+* Do users know what they're signing up for?
+* Emergent Tech and Privacy
+
```

Comment:
Be sure to add a closing activity:  Final share?  Each student provides their final definition of privacy and why it is important?